### PR TITLE
Include species in markdown output

### PR DESCRIPTION
**Pull request by CheshireSwift**
I noticed that it's hard to work out the species of the team members when you come back after a while, so wanted to include it in the canvas output.

Needed to add the species index mapping (which is nowhere near dex order - might be due a comment...) so that we can display it.

I made a unit test file so I could test this without going through the full setup for test integration with Slack, but I'm pretty certain it either isn't in the right place or should be excluded from packages or something - please advise on appropriate location/changes (or I can just delete it if it's not wanted).

### CODE REVIEW

File: src/integration/byte_mappings.py [hunk 1]
```diff
@@ -24,6 +24,201 @@
...
+    "Growlithe",
+    "Onix",
+    "Fearow",
+    "Pidgey",
+    "Slowpoke",
+    "Kadabra",
+    "Graveler",
+    "Chansey",
+    "Machoke",
+    "Mr",
```
Comment (AyaPK) [inline src/integration/byte_mappings.py]: Mr who 👀

File: src/test.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import unittest
+from integration.gen_1_pokemon import Pokemon
+
+class TestPokemonDisplay(unittest.TestCase):
```
Comment (AyaPK) [inline src/test.py]: tbh it's nice to have tests here, we slacked on them pretty hard in initial setup, thanks!

**Comment by AyaPK** [comment]
Local testing against a set-up slackbot looks good.

image

File: src/test.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
+import unittest
```
Comment (AyaPK) [inline src/test.py]: I think the test should probably live in `slack-plays-pokemon/tests/test_gen_1_pokemon.py` if you could move it there! :D

File: src/integration/byte_mappings.py (hunk 1 above)
Comment (CheshireSwift) [reply src/integration/byte_mappings.py]: Curses. I caught Farfetch'd, but missed this 😁

File: src/test.py (hunk 3 above)
Comment (CheshireSwift) [reply src/test.py]: Done. Turns out I needed a lil test infrastructure in `tests/__init__.py` to make imports happy.

Also updated the README with the command to execute them 🙂

**Review by AyaPK — APPROVED** [review]

**Pull request merged**